### PR TITLE
docs: update 1.5.4 changelog

### PR DESCRIPTION
**Pull request by alexmt**

### CODE REVIEW

**Review by dthomson25 — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 # Changelog
 
-## v1.5.3 (Unreleased)
+## v1.5.5 (2020-05-14)
```
Comment (jannfis) [inline CHANGELOG.md:3]: Hm, I see no release `v1.5.5` yet? :)

**Review by jannfis — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (alexmt) [reply CHANGELOG.md:3]: it is almost ready :)

Comment (alexmt) [reply CHANGELOG.md]: released https://github.com/argoproj/argo-cd/releases/tag/v1.5.5

**Review by jannfis — APPROVED** [review]

**Pull request merged**